src/test/java/org/folio/rest/impl/ReceivingHistoryTest.java

```diff
@@ -137,6 +137,7 @@ private void verifyFields(Piece piece, PoLine poLine, PurchaseOrder purchaseOrde
     assertEquals(poLine.getPurchaseOrderId(), receivingHistory.getPurchaseOrderId());
     assertEquals(purchaseOrder.getDateOrdered(), receivingHistory.getDateOrdered());
     assertEquals(poLine.getOrderFormat().value(), receivingHistory.getPoLineOrderFormat().value());
```

Comment:
As per final discussion with UI team, please remove order format as they have to get PO Line for check-in flow anyway

Reply:
Done